.github/workflows/package.yml

```diff
@@ -103,26 +101,105 @@ jobs:
           echo "UPDATE_LOGFILE=update_log.txt" >> $GITHUB_ENV
           echo "NEW_BRANCH=${{ github.event.inputs.working_branch }}" >> $GITHUB_ENV
 
-      # Fetch all the zip files from previous run
-      - name: Fetch Desktop Artifacts
+      # Fetch all the zip files from previous run (OLD)
+      - name: Fetch iOS Artifacts (OLD)
+        uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == '0'
+        with:
+          workflow: 'ios.yml'
+          run_id: ${{ github.event.inputs.download_ios_run }}
+          name: ios_unity
+          path: built_artifect
+      
+      - name: Fetch Android Artifacts (OLD)
+        uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == '0'
+        continue-on-error: true
+        with:
+          workflow: 'android.yml'
+          run_id: ${{ github.event.inputs.download_android_run }}
+          name: android_unity
+          path: built_artifect
+
+      - name: Fetch Linux Artifacts (OLD)
         uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == '0'
+        continue-on-error: true
         with:
           workflow: 'sdk_build.yml'
-          run_id: ${{ github.event.inputs.download_desktop_run }}
+          run_id: ${{ github.event.inputs.download_linux_run }}
+          name: linux_unity
           path: built_artifect
 
-      - name: Fetch iOS Artifacts
+      - name: Fetch MacOS Artifacts (OLD)
         uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == '0'
+        continue-on-error: true
         with:
-          workflow: 'ios.yml'
+          workflow: 'sdk_build.yml'
+          run_id: ${{ github.event.inputs.download_macos_run }}
+          name: macos_unity
+          path: built_artifect
+
+      - name: Fetch Windows Artifacts (OLD)
+        uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == 'false'
+        continue-on-error: true
+        with:
+          workflow: 'sdk_build.yml'
+          run_id: ${{ github.event.inputs.download_windows_run }}
+          name: windows_unity
+          path: built_artifect
+
+      # Fetch all the zip files from previous run (NEW)
+      - name: Fetch Android Artifacts (NEW)
+        uses: dawidd6/action-download-artifact@v2
+        if: github.event.inputs.use_new_build == '1'
+        continue-on-error: true
+        with:
+          workflow: 'build_starter.yml'
```

Comment:
If the only difference between the "OLD" and "NEW" is the workflow name, could that be set in a variable, and then we wouldn't need to have each duplicated?